crates/parcel_plugin_rpc/src/rpc_conn_message.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::mpsc::Sender;
+
+pub enum RpcConnectionMessage {
+  Ping {
+    response: Sender<Result<(), String>>,
+  },
+}
+
+impl RpcConnectionMessage {
+  pub fn get_id(&self) -> u32 {
+    match self {
+      Self::Ping { response: _ } => 0,
+    }
+  }
+}
```

Comment:
adding an index to the enum is useful for casting the event type into JavaScript